packages/next/src/server/app-render/metadata-insertion/create-server-inserted-metadata.tsx

```diff
@@ -6,7 +6,15 @@ import {
 } from '../../../shared/lib/server-inserted-metadata.shared-runtime'
 import { renderToString } from '../render-to-string'
 
-export function createServerInsertedMetadata() {
+/**
+ * For chromium based browsers (Chrome, Edge, etc.) and Safari,
+ * icons need to stay under <head> to be picked up by the browser.
+ *
+ */
+const REINSERT_ICON_SCRIPT = `\
```

Comment:
what's the point of this slash?

Reply:
it's nothing but just be able to let you start the string in the new line for appearance 😂
<img width="107" alt="image" src="https://github.com/user-attachments/assets/ebf46fe4-654b-4b47-ba60-07f96dcb7514" />

Reply:
TIL